src/main/java/edu/hm/hafner/grading/FileSystemToolParser.java

```diff
@@ -38,7 +51,17 @@ public Report readReport(final ToolConfiguration tool, final FilteredLog log) {
         var analysisParser = parser.createParser();
         for (Path file : REPORT_FINDER.find(log, displayName, tool.getPattern())) {
             var report = analysisParser.parse(new FileReaderFactory(file));
-            total.addAll(report);
+            var baseline = Baseline.fromString(tool.getBaseline());
+
+            var marker = new IssuesInModifiedCodeMarker();
+            marker.markIssuesInModifiedCode(report, modifiedLines);
```

Comment:
It makes sense to filter by modified files as well. I created an issue [JENKINS-76222 - Add support for modified files in report filtering](https://issues.jenkins.io/browse/JENKINS-76222).

Reply:
Now released as https://github.com/jenkinsci/analysis-model/releases/tag/v13.12.0